conf/agent/agent.conf

```diff
@@ -31,4 +33,10 @@ plugins {
         plugin_data {
         }
     }
+
+    SvidPublisher "aws_secretsmanager" {
```

Comment:
Changes in agent.conf should be removed from this branch. We should update agent_full.conf to have an example of the new configured plugin.